wsl/os/almalinux9/FAQ.md

```diff
@@ -8,7 +8,7 @@ Execute the following command:
 sudo dnf module switch-to php:remi-{major}.{minor} -y
 ```
 
-where `{major}.{minor}` is one of the supported PHP versions: `8.3`, `8.2`, `8.1`, `8.0` and `7.4`.
+where `{major}.{minor}` is one of the supported PHP versions: `8.4`, `8.3`, `8.2`, `8.1`, `8.0` and `7.4`.
```

Comment:
remove 8.0 from this list, not supported

Reply:
and remove 7.4 too

Reply:
Removed from both distros.